docs/en/ingest-management/elastic-agent/running-on-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -67,8 +67,22 @@ To specify different destination/credentials, change the following parameters in
 <5> The basic authentication username used to connect to {kib} and retrieve a `service_token` to enable {fleet}.
 <6> The basic authentication password used to connect to {kib} and retrieve a `service_token` to enable {fleet}.
 
-Refer to <<agent-environment-variables>> for all available options.
+If you need to run {fleet-server} as well, adjust the `docker run` command above by adding these environment variables:
+
+[source,yaml]
+------------------------------------------------
+- name: FLEET_SERVER_ENABLE
+  value: "true" <1>
+- name: FLEET_SERVER_ELASTICSEARCH_HOST
+  value: "<elasticsearch-host>" <2>
+- name: FLEET_SERVER_SERVICE_TOKEN
+  value: "<service-token>" <3>
+------------------------------------------------
+<1> Set to `1` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```

Comment:
```suggestion
<1> Set to `true` to bootstrap {fleet-server} on this {agent}. This automatically forces {fleet} enrollment as well.
```
I just noticed that the description doesn't match the example. @michel-laterman Should `FLEET_SERVER_ENABLE` be set to `true` or to `1`, or are both okay?

Reply:
Both values are accepted and will be parsed as a bool `true` value, so I guess we should be consistent with whatever we have been using in the docs